add-ons/ansible-service-broker/README.md

```diff
@@ -1,71 +1,84 @@
-# Ansible Service Broker Add-on
-
+Ansible Service Broker Addon
+======================
 
 This addon installs the [Ansible Service Broker](https://github.com/openshift/ansible-service-broker)
 
-## Prerequisites
+# Prerequisites
+
+This addon is designed for an RBAC enabled Openshift cluster of `>=3.7`. Minishift can be configured to
+deploy 3.7 with the following command:
+
+```
+$ minishift config set openshift-version v3.7.1
+```
 
 Minishift/CDK must be deployed with the Service Catalog enabled
```

Comment:
Should we specify `CDK` here ? Not sure. /cc @LalatenduMohanty

Reply:
@budhrg I think it is fine to mention as it is intended for both. But the minimum criteria for any add-on in this repository is that it would work with upstream `minishift` repository.